docker/nginx/conf.d/include/track-registry

```diff
@@ -4,15 +4,14 @@ log_by_lua_block {
     if os.getenv("ACCOUNTS_ENABLED") ~= "true" then return end
 
     if ngx.status == ngx.HTTP_OK or ngx.status == ngx.HTTP_NOT_FOUND then
-        local http = require("socket.http")
+        local httpc = require("resty.http").new()
```

Comment:
I would call this `client`. `httpc` immediately sets off my alarm bells.

If I forget to switch back from cyrillic to latin layout and I try to change `http` to `https` it will end up `httpс`. :)

Reply:
I used naming convention from the official documentation https://github.com/ledgetech/lua-resty-http#single-shot-request - it was easier to manage copy-pasting code :)

Reply:
I see. Well, if that's their recommended style then that's fine. Let's use their recommendation.